lib/fat_zebra/authorise.rb

```diff
@@ -0,0 +1,49 @@
+module FatZebra
+  ##
+  # == FatZebra \Authorise
+  #
+  # Manage 3DS authorisation for the API
+  #
+  # * token
+  # * authorise
+  #
+  class Authorise < APIResource
+    @resource_name = 'authorise'
+
+    validates :card_number, required: true, on: :authorise
+    validates :card_expiry, required: true, on: :authorise
+    validates :threeds, required: true, on: :authorise
+
+    class << self
+
+      ##
+      # Returns a JWT token for the 3DS authorisation process
+      #
+      # @param [Hash] params
+      # @param [Hash] options for the request, and configurations (Optional)
+      #
+      # @return [FatZebra::Authorise]
+      def token(params = {}, options = {})
```

Comment:
What do you think of renaming this endpoint to `jwt` or `jwt_token` to avoid confusion with credit card "tokens"?

Reply:
:+1